### DISCUSSION TITLE
Is the connection migration support?

### DISCUSSION THREAD

**Discussion by machao0322** [microsoft/msquic#1496] [Q&A] [answered]
Is the connection migration support? Thanks!

**Comment by nibanks** [comment, marked as answer]
It is generally supported on the server side. On the client, we don't support explicit migration to new IP addresses yet.

**Comment by machao0322** [comment]
Glad to hear that the connection migration is supported!
Thank you very much for your reply.&nbsp;Sincerely,
Tengchao ma




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "Nick ***@***.***&gt;; 
发送时间: 2021年4月20日(星期二) 晚上10:37
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [microsoft/msquic] Is the connection migration support? (#1496)





 
It is generally supported on the server side. On the client, we don't set support explicit migration to new IP addresses yet.
